solace-spring-boot-autoconfigure/solace-java-spring-boot-autoconfigure/pom.xml

```diff
@@ -25,12 +25,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-configuration-processor</artifactId>
-			<optional>true</optional>
```

Comment:
`spring-boot-configuration-processor` needs to stay optional as per Spring docs: https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html#appendix.configuration-metadata.annotation-processor.configuring

Reply:
Fixed.